airflow/models/mappedoperator.py

```diff
@@ -774,6 +776,7 @@ def unmap(self, resolve: None | Mapping[str, Any] | tuple[Context, Session]) ->
         from airflow.serialization.serialized_objects import SerializedBaseOperator
 
         op = SerializedBaseOperator(task_id=self.task_id, params=self.params, _airflow_from_mapped=True)
+        # TODO Should i somehow re-assign it here as well?
```

Comment:
Up to you; nothing else than OL needs the information (as far as I know).

Reply:
Ok, for now added the `upstream_task_ids` that is not being serialised, the downstream is already there.